api/app/src/providers/withings.ts

```diff
@@ -88,6 +88,25 @@ export class Withings extends Provider implements OAuth2 {
     return JSON.stringify(response.data.body);
   }
 
+  async subscribeToWebhooks(token: string, callbackUrl: string) {
+    const access_token = JSON.parse(token).access_token;
+    const webhookCategories = [1, 2, 4, 16, 44, 54, 58];
```

Comment:
these are the same as `api/app/src/command/webhook/withings.ts` right? let's use the same consts so we don't have to update them in 2 places?